src/Microsoft.DotNet.Interactive.Http/HttpKernel.cs

```diff
@@ -53,6 +54,8 @@ public class HttpKernel :
         _responseDelayThresholdInMilliseconds = responseDelayThresholdInMilliseconds;
         _contentByteLengthThreshold = contentByteLengthThreshold;
 
+        KernelCommandEnvelope.RegisterCommand<ClearValues>();
```

Comment:
As long as the underlying implementation is static, this can be done in a static constructor.